set.mm

```diff
@@ -250723,6 +250723,22 @@ such that every prime ideal contains a prime element (this is a
     $d x y z A $.  $d x y z L $.  $d x y z S $.  $d x y z W $.
     issubassa.s $e |- S = ( W |`s A ) $.
     issubassa.l $e |- L = ( LSubSp ` W ) $.
+    $( A subring that is also a subspace is a subalgebra.  The key theorem is
```

Comment:
How does this comment come about?  I don't find a matching theorem from MC from 7-Jan-2015.

Reply:
It's a modified version of the comment from ~ issubassa where instead of a biconditional, the sentences states one direction only.

The part about the key theorem would've been like a note to past me that would explain how LSubSp relates to AssAlg (although now looking at it again, this theorem could just use A e. LMod instead of A e. L)